contracts/PetPark.sol

```diff
@@ -1,7 +1,201 @@
-//SPDX-License-Identifier: Unlicense
+// SPDX-License-Identifier: Unlicense
 pragma solidity ^0.8.0;
 
+contract PetPark
+{
+    // -- SCAFFOLDING -------------------------
```

Comment:
If adding comments, the better way would be to use [natspec](https://docs.soliditylang.org/en/v0.5.10/natspec-format.html)